src/main/java/de/dentrassi/rpm/builder/YumMojo.java

```diff
@@ -235,6 +237,47 @@ private void addSinglePackage(final Path path, final Context context) throws IOE
             rpmInformation = RpmInformations.makeInformation(ris);
         }
 
+        // Retrieve all the files contained in the rpm (including symlink)
+        List<String> providedFiles = Lists.newArrayList();
+        try (RpmInputStream ris = new RpmInputStream(Files.newInputStream(path))) {
+            ris.getPayloadHeader();
+            ris.getSignatureHeader();
+
+            final CpioArchiveInputStream cpio = ris.getCpioStream();
```

Comment:
Wouldn't it make sense to use a `try-with-resource` for `cpio` too?

Reply:
Hello, I'm working with ThorAsgardDev. Sure, try-with-resources should work. He did it like that because it is the way use in https://github.com/eclipse/packager/blob/master/rpm/src/main/java/org/eclipse/packager/rpm/info/RpmInformations.java (line 98).

Reply:
Ah, I see … I guess that is some legacy that should also be cleaned up at some point :grin: 

I think we can do better here. Unless there are technical reasons, I would appreciate if you could make this quick change.